recommender/utils.py

```diff
@@ -181,25 +179,26 @@ def load_examples() -> Dict:
         "target_users": [1, 2, 3],
     }
 
+    # TODO: Commented, as it yet again breaks the application xd - migrations this time
```

Comment:
Why does it break our app?

Reply:
1. This code is run when the flask app initialization is run. 
2. When you use the flask CLI (as we are in the new migration commands) the flask app initialization is run.
3. This code uses the DB and some models.
4. Since it uses the DB and some models, and the migrations have not been run yet (the flask app is initializing) the are errors thrown (like missing "column" name which should be renamed by the migration.(